22 blog post I have a question, but I don't have WhatsApp! (#51)
wpaccessibility/wp-a11y-docs: _posts/2025-08-27-offer-multiple-ways-to-contact-you.md

---
layout: post
title: "I have a question, but I don’t have WhatsApp!"
permalink: /blog/2025/offer-multiple-ways-of-contact/
author: Rian Rietveld

> categories: contact, multiple ways

Reviewer: For EAA (and consumer law btw) customer service needs to be reachable in multiple ways, so I feel this makes sense on top of contact.

> categories: contact, multiple ways

Author: I removed the category, it's in the content.

parent: Blog
excerpt: "It’s better to offer multiple ways to get in touch, so people can choose what suits them. Think about your visitors first, not just what’s easiest for you.

> "

Author: Good catch, thx.

date: 2025-08-27
---

> You want to reach out to the customer service, and they say  *“Send us a WhatsApp message.”* WHat if you don't use WhatsApp, or can make a phone call, or use the chatbox? If you offer multiple contact options, you're leaving no people (customers) behind.

Reviewer 2: ```suggestion
You want to reach out to the customer service, and they say  *“Send us a WhatsApp message.”* What if you don't use WhatsApp, or cannot make a phone call, or use the chatbox? If you offer multiple contact options, you're leaving no people (customers) behind.
```

> You want to reach out to the customer service, and they say  *“Send us a WhatsApp message.”* What if you don't use WhatsApp, or cannot make a phone call, or use the chatbox? If you offer multiple contact options, you're leaving no people (customers) behind.

Reviewer 3: Omit "the": "You want to reach out to customer service..."

{: .fs-6 .fw-300 }

> If you offer multiple contact options, you’re leaving no people (customers) behind.

Reviewer: I like the softer inclusive option

> If you offer multiple contact options, you’re leaving no people (customers) behind.

Author: I changed the text into:
By offering multiple contact options, you make sure every customer has a way to connect.

## Don’t make getting in touch their problem

Let’s say someone wants to ask a quick question through your website, but the only way to do it is by sending a WhatsApp message.

Sure, it’s convenient for you. One place to manage it all, and you reply when it suits you.

> Other examples:

Reviewer 3: Make this line flow more smoothly:

"There are many similar examples of this type of problem:"

Author: I changed the text into
"There are many similar examples"

- You only list the phone number of your helpdesk.
- You only offer a chat function, with a bot you hope can handle basic questions. And if not, it points people to the “FAQs”.
- You require people to come in person to renew their passport.
- You provide a clunky, hard-to-use form to submit a complaint, just because that’s how it’s set up in your customer information software.

In all these cases, you’re forcing your way of working on the people trying to reach you. And it’s worth remembering: some of them can’t even use these options.

> So, what are the ways to communicate? And what are the pros and cons?

Reviewer 3: I think this is clearer as "So, what are some other ways to communicate? What are the pros and cons of different methods?"

## Different ways to get in touch

### Social media and instant messaging

> Services like WhatsApp or X (formerly Twitter) can be useful, it’s widely used, low-barrier, and great for quick questions.

Reviewer 3: "Services like WhatsApp or X (formerly Twitter) can be useful. They're widely used, pose a low barrier to usage, and are great for quick questions."

> But don’t assume everyone uses it. Many of people are moving away from WhatsApp due to Meta’s privacy issues. Some prefer Signal or don’t want to share personal stuff over WhatsApp at all.

Reviewer 3: "But don't assume everyone uses [them]. Many people are moving away from social media services due to privacy issues. Some prefer more private services, like Signal, or don't want to share personal information over your chosen service."

- This has a lot of changes; but I think it's good to make this broader. It's not *just* Meta's privacy issues; there are privacy issues in many of these services.

So keep in mind: not everyone uses what you prefer.

### Phone call

> Many people prefer a phone call, real person, instant answers. Definitely offer it as an option if possible.

Reviewer 3: "Many people prefer a phone call to reach a real person and get instant answers. You should definitely offer it as an option if possible."

User research shows that people with low literacy prefer calling first:

> “I always try calling first. If that doesn’t work, I’ll check the website to find another way.”

> But for some, like deaf, hard-of-hearing, shy, autistic, or stuttering people, phone calls can be tough. Same goes for people who struggle your native language. Image you're in a noisy train and don’t want to talk about private stuff.

Reviewer 2: ```suggestion
But for some, like deaf, hard-of-hearing, shy, autistic, or stuttering people, phone calls can be tough. The same goes for people who struggle with your native language. Image you're in a noisy train and don’t want to talk about private stuff.
```

Some callers even get hung up on if they take too long to respond. Wouter Honselaar, who stutters, said in the interview [I am stronger than stuttering](https://www.a11y-collective.com/blog/interview-with-wouter-i-am-stronger-than-stuttering/):

> “Once, someone ended the call because they thought the line had gone dead.”

Also, phone calls often only work during office hours. If you work during those hours and can’t make personal calls, it’s not an option.

> I asked on LinkedIn my deaf followers:

Reviewer 2: ```suggestion
I asked my deaf followers on LinkedIn:
```

> I asked my deaf followers on LinkedIn:

Reviewer: Who is the I in this sentence? Because we don't mention the original author, right? Only the original post. Maybe rewrite this?

Author: ah, I will changes the I into Rian

> I asked my deaf followers on LinkedIn:

Reviewer: And in this sentence as well

Author: same as previous comment

[Marie Van Driessche](https://www.linkedin.com/in/marie-van-driessche/) replied:

> “I use a [video relay service](https://en.wikipedia.org/wiki/Video_relay_service) via the Tolkcontact app. So I get a translator to call them for me. It works, but I have to arrange it all myself. It’d be nice if email or WhatsApp were options.”

[Meryl Evans](https://www.linkedin.com/in/meryl/) added:

> “Offer text-based options like email, SMS, live chat with a real person, or a video relay. Let people choose what contact details they share. Don’t make phone numbers required.”

### Email

The nice thing about email is that you can send one whenever you want.

> But not everyone finds it easy to write an email, especially if they’re not confident in your langauge, have dyslexia or low literacy. Some people may need help just to write a message.

Reviewer 3: langauge => language

### Contact Form

Contact forms are handy, you ask exactly what you need, and users can fill it out in an organized way.

> But forms can be a real barrier for people who aren’t great with tech. Especially if the form is long or complicated. Always test your forms with real users, including people with disabilities.

Reviewer 3: "But forms can be a real barrier for people who aren’t great with tech, especially if the form is long or complicated."

> But forms can be a real barrier for people who aren’t great with tech, especially if the form is long or complicated. Always test your forms with real users, including people with disabilities.

Reviewer: I suppose we need to determine how we want to address the target audience, disability first or person first, etc. Autistic versus person with autism. Deaf (capitalized) and hard-of-hearing, or people with auditory disabilities. New discussion issue?

If someone is blind, visually impaired, or just not tech-savvy, a poorly set up and coded form can be a nightmare.

So, make sure it’s easy to use and it’s accessible.

### Chat box

Quick question? Don’t want to call? A chat box sounds perfect.

> Until you get stuck with a robot that can’t help and just tells you to “check the FAQs” or to call someone. Instead of helping, you’re fobbing people off.

Reviewer 2: ```suggestion
Until you get stuck with a robot that can’t help and tells you to “check the FAQs” or to call someone. Instead of helping, you’re fobbing people off.
```

> Deaf and hard-of-hearing people say chat can be a great alternative to phone calls, as long as it’s a real person.

Reviewer 2: ```suggestion
Deaf and hard-of-hearing people say that chat can be a great alternative to phone calls, as long as it involves a real person.
```

### In person contact

Sometimes, like when renewing a passport, you have to show up in person to confirm your identity.

But what if your health makes that impossible? Then what?

> Therefore, make sure you offer an alternative. As a municipality you can offer a home visit to confirm the identity of a resident.

Reviewer 3: "Make sure you offer an alternative. As a municipality you can offer a home visit to confirm the identity of a resident."

### A letter

As a bonus option, sending an old-school paper letter is helpful if you're not computer-savvy. It's definitely an option as an alternative to all that digital communication.

## Summary: communication matters

It’s important to offer multiple ways for people to get in touch. 

{: .example }
You could ask for example:

> “Would you like us to call you back or send an email instead?”

Reviewer 3: And don't just ask for their preferences; respect them.

(This is a pet peeve for me: it's *shockingly* common in the US for companies to have a way for you to specify your preference, but I've never yet had a company actually follow it. I always say I prefer email, they always call me.)

> It’s important to offer multiple ways for people to get in touch. And don’t just ask for their preferences; respect them.

Reviewer 2: ```suggestion
It’s essential to provide multiple options for people to contact you. And don’t just ask for their preferences; respect them.
```
